### ISSUE TITLE
python highlight error

### ISSUE THREAD

**Issue by aeschli** [MagicStack/MagicPython#160]
_From @baby5 on September 5, 2018 3:51_

 

- VSCode Version: 1.26.1
- OS Version: win10 1803 17134.228

Steps to Reproduce:
![example](https://user-images.githubusercontent.com/12880800/45069904-6311e180-b101-11e8-8578-3166d6681f27.png)
1.related import "important_module" should not be highlight

Does this issue occur when all extensions are disabled?: Yes

_Copied from original issue: Microsoft/vscode#57930_

**Comment by aeschli** [comment]
_From @vscodebot[bot] on September 5, 2018 3:51_

(Experimental duplicate detection)
Thanks for submitting this issue. Please also check if it is already covered by an existing one, like:
- [Python syntax highlighting no longer works (#55318)](https://www.github.com/Microsoft/vscode/issues/55318)

**Comment by aeschli** [comment]
_From @mjbvz on September 5, 2018 6:40_

Please share the text of the code along with screenshots

**Comment by aeschli** [comment]
_From @baby5 on September 5, 2018 8:31_

@mjbvz `from .important_module import something`

**Comment by baby5** [comment]
@aeschli hello,thanks for your work.
i have seen the same [issues](https://github.com/MagicStack/MagicPython/issues/133) and it has been [fixed](https://github.com/MagicStack/MagicPython/commit/7c43d1cb668f349b292356433272b42dab5de69d)
i think vscode need update for this.

**Comment by aeschli** [comment]
Oh, we currently can't update, blocked by #150.

Marking as duplicate of #133 then.
